packages/@okta/vuepress-site/docs/concepts/brands/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Brands
+meta:
+  - name: description
+    content: Okta brands allow you to customize the look and feel of pages and templates, such as the Okta-hosted sign-in page, error pages, email templates, and the Okta End-User Dashboard.
+---
+
+# Brands
+
+## What are Okta brands?
+
+Okta brands allow you to customize the look and feel of pages and templates. You can customize the Okta-hosted sign-in page, error pages, email templates, and the Okta End-User Dashboard.
+
+Each org starts off with Okta default branding. You can upload your own assets (colors, background image, logo, and favicon) to replace Okta default brand assets. You can then publish these assets directly to your pages and templates.
+
+## What is multibrand customization?
+
+> **Note:** Multi-brand customization is available on both Okta Identity Engine and Okta Classic Engine.
+
+Multibrand customizations allow customers to use one org to manage multiple brands and multiple custom domains. This drastically simplifies multi-tenant architectures, where customers used to have to create multiple orgs to satisfy branding requirements. Multibrand customizations allow orgs to create up to three custom domains (more upon request), which can be mapped to multiple sign-in pages, multiple sets of emails, error pages, and multiple versions of the Okta End User Dashboard.
+
+In addition, multibrand allows you to customize many other settings per brand:
+
+- Okta loading page
+- Sign-In Widget version
+- Sign-in page labels
+- Default app for the Sign-In Widget
+- Display language
+- Sign-out page custom URL
+
+Okta Verify and the Okta plugin support multibrand. Use the following minimum versions for compatibility:
+
+- Plugins: v6.16.0
+- Okta Verify for Android: AOV 7.10.0
+- Okta Verify for iOS: iOV 7.12.0
+- Okta Verify for macOS: MOV 3.10
+- Okta Verify for Windows: v3.10
+
+### Multibrand customizations and custom domains
+
+You can create up to three custom domains with multibrand customizations. Increase your limit to 200 custom domains by contacting support.
+
+You can only visit a branded touchpoint (such as a logo, color, favicon, or an image applied to a page or email) once you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them. See [Custom domains](/docs/guides/custom-url-domain/main/#about-okta-domain-customization).
```

Comment:
I'm unsure about my edit here. I wanted to combine the first two sentences in some way, as they seemed repetitive......take this edit or leave it the same. :) 
```suggestion
You can only visit a branded touchpoint (such as a logo, color, favicon, or an image applied to a page or email) after you map to a custom domain. Create a brand and map it to a custom domain. Then, you can make further customizations, preview them, and publish them. See [Custom domains](/docs/guides/custom-url-domain/main/#about-okta-domain-customization).
```